enhancement! : remove all field label, field hint, field errors from all possible inputs (#1693)
strapi/design-system: BREAKING_CHANGES.md

This is the same component as `ToggleInput` and therefore you should use that instead.

### Removed field wrapper components from inputs

> We are streamlining inputs by removing unnecessary components like labels, hints, and error messages, focusing solely on the inputs themselves. This simplifies the structure and reduces DOM size, especially in cases where only aria-labels are needed. Users can still use Field components they used alongside inputs.

Reviewer: ```suggestion
We're making inputs more composable by breaking down the pieces from components like labels, hints, and error messages, focusing solely on exporting the inputs themselves. This simplifies the structure and reduces DOM size, especially in cases where only aria-labels are needed. Users can still use Field components they used alongside inputs.
```

I'd then do another `diff` example to demonstrate before & after

### Components renaming

> ToggleInput is renamed to just Toggle.

Reviewer: ```suggestion
`ToggleInput` has been renamed to `Toggle`.
```

you could also add this (i dont know how to include it in the suggestion:

```diff
- import { ToggleInput } from '@strapi/design-system'
+ import { Toggle } from '@strapi/design-system'
```

This is the same component as `ToggleInput` and therefore you should use that instead.

### Removed field wrapper components from inputs

We're making inputs more composable by breaking down the pieces from components like labels, hints, and error messages, focusing solely on exporting the inputs themselves. This simplifies the structure and reduces DOM size, especially in cases where only aria-labels are needed. Users can still use Field components they used alongside inputs.

```ts

> // before
>  return (
>       <Field hint={hint} error={error} id={generatedId} required={required}>
>         <Flex direction="column" alignItems="stretch" gap={1}>
>           {label ? (
>             <FieldLabel onClick={handleFieldLabelClick} action={labelAction}>
>               {label}
>             </FieldLabel>
>           ) : null}
>           <SingleSelectInput
>             label={label}
>             id={generatedId}
>             ref={composedTriggerRefs}
>             required={required}
>             {...props}
>           />
>           <FieldHint />
>           <FieldError />
>         </Flex>
>       </Field>
>  );
> 
> // after
> return (
>           <SingleSelect
>             aria-label={label}
>             id={generatedId}
>             {...props}
>           />
>     );

Reviewer: ```suggestion
// before
return <SingleSelect label={label} error={error} required={required} value={value} onChange={handleChange} />;

// after
return (
  <Field>
    <FieldLabel>{label}</FieldLabel>
    <SingleSelect label={label} error={error} required={required} value={value} onChange={handleChange} />
    <FieldError />
    <FieldHint />
  </Field>
);
```
